Support for big definition overrides

BrunoBernardino opened the pull request:
Using an RPC call to get those.

Also adds support for building a boilerplate zip with the current "local" changes, vs the latest npm package.

This needs https://github.com/zapier/zapier/pull/17392 to ship before merging this PR.

Related to [PDE-166](https://zapierorg.atlassian.net/browse/PDE-166).

Commit:
Support for big definition overrides

Using an RPC call to get those.

Also adds support for building a boilerplate zip with the current "local" changes, vs the latest npm package.

Commit:
Remove unnecessary parse

eliangcs reviewed (approved):
Tested with the backend. It works! 👍

Commit:
Merge branch 'master' into feature-support-big-override

Commit:
Support a hash instead of a boolean for big definitions.

Cache big definitions, to reduce RPC calls.

Add tests.

Fix some tests.

Improve README for integration tests mention.

BrunoBernardino commented:
@eliangcs would you mind checking again? I've got definition caching working now!

eliangcs commented on integration-test/integration-test.js:
```diff
@@ -56,7 +57,7 @@ runLocally.testName = 'runLocally';
 const doTest = runner => {
   describe(`${runner.testName} integration tests`, () => {
-    it('should return data from app function call', done => {
+    it('should return data from app function call', () => {
```
Nice style change!

eliangcs reviewed (approved):
The change looks good! I assume you already tested it so I didn't pull.
